config/vsphere-csi-driver/kustomization.yaml

```diff
@@ -3,7 +3,6 @@ kind: Kustomization
 namespace: "{{ .Release.Namespace }}"
 
 patchesStrategicMerge:
-  - patches/daemonset-images.yaml
```

Comment:
This patch makes image and tag configurable but I don't see the point in this as they will remain outdated if we don't go looking manually for the versions in the upstream repo ourselves. I think we should rely on the image tags used in each release.